### ISSUE TITLE
Site settings cannot be saved when theme color picker is enabled

### ISSUE THREAD

**Issue by scott1702** [silverstripe/cwp-agencyextensions#15]
When clicking 'Save' in the settings for the first time after enabling the theme color picker the user is presented with a validation error. The cause of this error in the 'Main font family' dropdown which doesn't seem to have a value by default. This can be avoided by opening the font dropdown and selecting a font, however this is not clear to the user unless they are in the 'Theme Options' tab where the error is shown.

**Comment by robbieaverill** [comment]
I can't reproduce this on CWP 2.2.0-rc1. Steps:

* `composer require cwp/cwp-recipe-kitchen-sink ./cwp22 2.2.0-rc1`
* `vendor/bin/sake dev/build flush=1`
* Add to `app/_config/mysite.yml`:
```yaml
SilverStripe\SiteConfig\SiteConfig:
  enable_theme_color_picker: true
```
* Go to `admin/settings?flush=1`
* Hit Save

I get a successful save.

I've lowered the impact rating. Can you provide more info to reproduce @scott1702?

**Comment by robbieaverill** [comment]
Ok I reproduced it:

* `vendor/bin/sake dev/build flush=1`
* Go to `admin/settings`
* Save
* Apply the config
* Add `?flush=1` to the query string
* Save again

See validation failure

**Comment by robbieaverill** [comment]
The problem is that before you've viewed the Theme Options tab, the form field has not been rendered by React yet. I've made a PR at #16 containing a temporary hidden input that will provide the field value before the React component has been rendered, to prevent validation failures.

**Comment by robbieaverill** [comment]
Fixed in #16 with a follow up patch in #17
